apps/contracts/contracts/Zengo.sol

```diff
@@ -15,13 +15,33 @@ contract ZengoDAO is PermissionsEnumerable, ContractMetadata {
     address public votingTokenAddress; // Address of the ERC20 voting token
     address public deployer;
 
+    // Modify the Design to facilitate current implementation
     struct Proposal {
-        uint256 id;
+        uint256 proposalId;
         string title;
-        string description;
-        address submitter;
+        string proposalDescription;
+        string proposalType;
+        address proposer;
+        Evidence proposerEvidence;
+        Vote[] votingIterations;
+        // requires funding
+        bool isEligibleForFunding;
         bool isVerified;
-        uint256 votes;
+    }
+
+    struct Vote {
+        uint256 proposalId;
+        uint256 totalVotes;
```

Comment:
the `totalVotes` represents the total votes that will be distributed among the moderators?

Reply:
no total vote of that iteration